### ISSUE TITLE
Test Issue

### ISSUE THREAD

**Issue by BlackSquirrelz** [makeitr3al/XBorg#3]
/gpt

**Comment by 3ru** [comment]
@makeitr3al 
Two ymls are required, one for the command execution workflow and one for the commit workflow.

**Comment by 3ru** [comment]
@makeitr3al 

So, Is there now an automatic translation PR created when there is a change in the EN directory?

**Comment by makeitr3al** [comment]
@3ru 
Yes and much more important: there is no PR after a commit in any directory which is NOT /EN 😃 

Thank you SO much for your help, you have no idea how long I've searched a solution for this. 😅

Is there any possibility to configure the used version of ChatGPT?

**Comment by 3ru** [comment]
@makeitr3al 

Hype!

That is not possible at this time, but I would like to address it soon.

**Comment by makeitr3al** [comment]
@3ru 
Alright no worries 😀 

What version do you use now?

**Comment by 3ru** [comment]
@makeitr3al 
Currently GPT-3.5-turbo.

GPT-4 is more accurate, but I believe it is only available to the general public after the end of this month.

**Comment by makeitr3al** [comment]
@3ru 
Nice, so then I can reach you again when version 4 is out or do you need some prep time?

**Comment by 3ru** [comment]
@makeitr3al 

I don't think it will take more than a day to prepare!
I will be ready to respond as soon as possible once 4 is available to the public.

**Comment by makeitr3al** [comment]
/gpt-translate EN/04-or-team/meet-the-team.md DE/04-or-team/meet-the-team.md German

**Comment by makeitr3al** [comment]
/gpt-translate EN/04-or-team/meet-the-team.md DE/04-or-team/meet-the-team.md German

**Comment by makeitr3al** [comment]
/gpt-translate EN/04-or-team/meet-the-team.md CN/04-or-team/meet-the-team.md Chinese

**Comment by makeitr3al** [comment]
/gpt-translate EN/04-or-team/meet-the-team.md DT/04-or-team/meet-the-team.md Dutch

**Comment by makeitr3al** [comment]
/gpt-translate EN/04-or-team/meet-the-team.md JP/04-or-team/meet-the-team.md Japanese

**Comment by makeitr3al** [comment]
/gt EN/02-or-protocol/gaming-application-network/*.md AL/02-or-protocol/gaming-application-network/*.md Albanian

**Comment by makeitr3al** [comment]
/gt EN/07-or-organization/*.md AL/07-or-organization/*.md Albanian
